app/src/main/java/org/mozilla/fenix/settings/HomeSettingsFragment.kt

```diff
@@ -143,7 +143,6 @@ class HomeSettingsFragment : PreferenceFragmentCompat() {
         }
 
         requirePreference<SwitchPreference>(R.string.pref_key_history_metadata_feature).apply {
-            isVisible = FeatureFlags.historyMetadataUIFeature
```

Comment:
Nit: I think we may need to update also the UI as the visibility is hidden by default.
https://github.com/mozilla-mobile/fenix/blob/8589aec0a0c0c4ac5d8aeb1bde61e96460b504c5/app/src/main/res/xml/home_preferences.xml#L27-L29